python/gammaloop/base_objects/model.py

```diff
@@ -359,14 +359,20 @@ def is_ghost(self) -> bool:
     def is_massive(self) -> bool:
         return self.mass.value is None or abs(self.mass.value) > 0.
 
+    @staticmethod
+    def santize_texname(texname: str) -> str:
```

Comment:
I think there is a typo here, assuming that sanitize_texname was the intended function name :)

Reply:
Yeap, good catch!

Reply:
@GraDje13 Could you try a build from scratch on your system and it worked?